.cookiecutter/cookiecutter.json

```diff
@@ -6,9 +6,14 @@
     "Dockerfile",
     "lms/app.py",
     "tests/unit/lms/app_test.py",
-    "lms/pshell.py",
+    "lms/db.py",
     "lms/migrations/env.py",
     "tests/functional/app_test.py",
+    "tests/pytest_plugins/db.py",
+    "tests/pytest_plugins/pyramid.py",
+    "tests/unit/pytest_plugins/db.py",
+    "tests/unit/pytest_plugins/pyramid.py",
+    "tests/functional/pytest_plugins/db.py",
```

Comment:
LMS has its own versions of all these fixtures. It might be nice to get LMS using the cookiecutter's version of these (or at least using them as bases to extend) but it'd probably take some work to figure that out. Punting on it for now.